lib/ansiblelint/__main__.py

```diff
@@ -74,6 +80,22 @@ def choose_formatter_factory(
     return r
 
 
+def hint_about_skips(matches: List["MatchError"]):
+    """Display information about how to skip found rules."""
+    msg = """\
+Do **not** file bugs for rule violation just because they do not apply for your own
+use-case. You can skip rules using a configuration like below:
```

Comment:
This message can be misleading. It might happen a new rule does not behave as expected. It's not a problem to skip a rule if it highlights a problem and the dev does not care.
It is a problem when a default rule reports false positive (see #966). Thus I would not discourage people to file bugs about rules implementation.

Reply:
@btravouillon Can you please suggest another sentence? The idea is indeed about discouraging users from creating bug reports right away. Obviosuly that genuine bugs are ok to file.

Reply:
I would suggest the following text:

``If you would prefer to skip this rule, then use the following configuration for ansible-lint:``